Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md

```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
 
 ![MonitoringLeftNavScreenShot](../../images/MonitoringLeftNavScreenShot121317.JPG)
 
-##### Events
+##### [Events](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/events/)
```

Comment:
Change this to `(Events.md)`

See the comment I made in https://github.com/CenturyLinkCloud/PublicKB/pull/2158